content/en/docs/reference/setup-tools/kubeadm/kubeadm-join-phase.md

```diff
@@ -0,0 +1,62 @@
+---
+title: kubeadm join phase
+weight: 90
+---
+In v1.14.0, kubeadm introduces the `kubeadm join phase` command with the aim of making kubeadm more modular. This modularity enables you to invoke atomic sub-steps of the bootstrap process.
```

Comment:
of the **node** bootstrap process

Reply:
Calling out node bootstrapping might lead to confusion with also support control plane join functionality. I'd suggest s/bootstrap/join/

Reply:
it's just a copy paste from `init` but i will change.